product-service/test/handlers/get-products-list.test.js

```diff
@@ -1,39 +1,57 @@
 import { getProductsList } from '../../src/handlers/get-products-list';
-
 import * as productRepository from '../../src/repositories/product-repository';
 import * as response from '../../src/services/response';
 
-import products from '../../resources/product-list.json';
+import { productMock } from '../mocks/product.mock';
+import { responseMock } from '../mocks/response.mock';
 
-global.console = { log: jest.fn() };
+const products = [
+  productMock(),
+  productMock(),
+  productMock(),
+]
 
 beforeEach(() => {
   productRepository.findProducts = jest.fn();
   response.ok = jest.fn();
   response.serverError = jest.fn();
+  global.console = {
+    info: jest.fn(),
+    error: jest.fn(),
+  };
 });
 
 afterEach(() => {
```

Comment:
do u really need this block? u create new mock before each run in beforeEach block. 

at least, u could use  https://jestjs.io/docs/jest-object#jestclearallmocks
```
afterEach(() => {
     jest.clearAllMocks();
  })
```